<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionProfileDataPanel</class>
 <widget class="QWidget" name="SectionProfileDataPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="sysRefDefinitionBox">
     <property name="title">
      <string>Sistema di riferimento</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QLineEditValue" name="yDLEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabelRichName" name="zDLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
z&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabelRichName" name="yDLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
y&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabelUnitMeasure" name="yDUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m]
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEditValue" name="zDLEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabelUnitMeasure" name="zDUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m]
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabelRichName" name="thetaDLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;
&lt;/head&gt;
&lt;body&gt;
θ&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabelUnitMeasure" name="thetaDUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m]
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEditValue" name="thetaDLEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="geometricDataDefinitionBox">
     <property name="title">
      <string>Proprietà geometriche</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="7" column="1">
       <widget class="QLabelUnitMeasure" name="SDz3DyDzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;5&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabelUnitMeasure" name="SDy2zDyDzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;5&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEditValue" name="SDy2zDyDzLEdit"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabelRichName" name="SDyz2DyDzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
∬ y&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt; z&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt; dA
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabelUnitMeasure" name="SDyz2DyDzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;5&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLineEditValue" name="SDyz2DyDzLEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabelRichName" name="IDyLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
I&lt;span style=&quot; vertical-align:sub;&quot;&gt;D,y&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabelUnitMeasure" name="AUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabelRichName" name="SDyLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
S&lt;span style=&quot; vertical-align:sub;&quot;&gt;D,y&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEditValue" name="SDyLEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabelUnitMeasure" name="SDzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;4&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEditValue" name="ALEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabelRichName" name="SDzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
S&lt;span style=&quot; vertical-align:sub;&quot;&gt;D,z&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabelUnitMeasure" name="SDyUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabelRichName" name="ALabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
A
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEditValue" name="SDzLEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabelRichName" name="SDy3DyDzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
∬ y&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt; &lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt; dA
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabelUnitMeasure" name="SDy3DyDzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;5&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEditValue" name="SDy3DyDzLEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabelRichName" name="SDz3DyDzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
∬ z&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt; dA
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEditValue" name="IDyzLEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabelRichName" name="IDyzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
I&lt;span style=&quot; vertical-align:sub;&quot;&gt;D,yz&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEditValue" name="IDzLEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabelUnitMeasure" name="IDyUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;4&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabelUnitMeasure" name="IDzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;4&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabelUnitMeasure" name="IDyzUMLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m&lt;span style=&quot; vertical-align:super;&quot;&gt;4&lt;/span&gt;]
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabelRichName" name="IDzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
I&lt;span style=&quot; vertical-align:sub;&quot;&gt;D,z&lt;/span&gt;
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEditValue" name="IDyLEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabelRichName" name="SDy2zDyDzLabel">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
∬ y&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;z&lt;span style=&quot; vertical-align:sub;&quot;&gt;D&lt;/span&gt; dA&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEditValue" name="SDz3DyDzLEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelRichName</class>
   <extends>QLabel</extends>
   <header>qlabelrichname.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditValue</class>
   <extends>QLineEdit</extends>
   <header>qlineeditvalue.h</header>
  </customwidget>
  <customwidget>
   <class>QLabelUnitMeasure</class>
   <extends>QLabel</extends>
   <header>qlabelunitmeasure.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
